src/components/uielements/poolStep/PoolStatus.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+
+import { Row } from 'antd'
+import BigNumber from 'bignumber.js'
+import { useIntl } from 'react-intl'
+
+import Label from '../label'
+import Trend from '../trend'
+import { PoolStatusWrapper } from './PoolStatus.style'
+
+type Props = {
+  amount: BigNumber
```

Comment:
For amounts we are using `BaseAmount` or `AssetAmount` from `asgardex-util` https://gitlab.com/thorchain/asgardex-common/asgardex-util/-/blob/master/src/types.ts#L19-20 I would suggest to use `AssetAmount` here. Checkout [`asset.test.ts`](https://gitlab.com/thorchain/asgardex-common/asgardex-util/-/blob/master/src/asset.test.ts) to see examples about how to use one of these ^.